azure-pipelines.yml

```diff
@@ -509,6 +509,50 @@ jobs:
     displayName: Upload artifacts for CW310
     condition: failed()
 
+- job: chip_earlgrey_cw340
```

Comment:
Could this be achieved using matrix? https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/jobs-job-strategy?view=azure-pipelines

Reply:
There is actually a template in the integrated repo, and it would be good to bring it over, tweak it, and apply it to all bitstream jobs. However, this PR is rather large already, hehe.

I plan to do that separately.